cmd/apiserver/server_test.go

```diff
@@ -0,0 +1,401 @@
+package main
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/sirupsen/logrus"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	calculationsv1 "github.com/vega-project/ccb-operator/pkg/apis/calculations/v1"
+	"github.com/vega-project/ccb-operator/pkg/client/clientset/versioned/fake"
+	v1 "github.com/vega-project/ccb-operator/pkg/client/clientset/versioned/typed/calculations/v1"
+	"github.com/vega-project/ccb-operator/pkg/util"
+)
+
+func TestCreateCalculation(t *testing.T) {
+	testCases := []struct {
+		id               string
+		jsonStr          []byte
+		calculationCount int
+		expected         []calculationsv1.Calculation
+	}{
+		{
+			id:               "no calculation",
+			calculationCount: 0,
+		},
+		{
+			id:               "one calculation gets created",
+			jsonStr:          []byte(`{"teff": "14390.0", "logG": "4"}`),
+			calculationCount: 1,
+		},
+	}
+
+	fakecs := fake.NewSimpleClientset()
+	var fakeClient v1.VegaV1Interface
+	fakeClient = fakecs.VegaV1()
```

Comment:
```suggestion
	fakeClient := fakecs.VegaV1()
```
and you can remove the line 41